deluge/core/core.py

```diff
@@ -54,6 +58,8 @@
 )
 from deluge.httpdownloader import download_file
 
+reactor = typing.cast(ReactorBase, twisted.internet.reactor)
```

Comment:
This isn't so nice. We import reactor quite a bit. If we enable mypy I think we should figure out some better way than doing this in every file.

Reply:
I think I figured this out. Made some stubs for twisted that I told mypy to load. Defined reactor in the twisted.internet.__init__.pyi stub. I _think_ that still lets mypy use the real definitions for all the other twisted modules.